ansible/modules/hashivault/hashivault_secret_engine.py

```diff
@@ -214,22 +227,55 @@ def hashivault_secret_engine(module):
     # create
     if changed and not exists and state == 'enabled' and not module.check_mode:
         if backend == 'kv' or backend == 'kv-v2':
-            client.sys.enable_secrets_engine(backend, description=description, path=name, config=config,
-                                             options=options)
+            client.sys.enable_secrets_engine(
+                backend,
+                description=description,
+                path=name,
+                config=config,
+                options=options,
+                seal_wrap=seal_wrap
```

Comment:
If seal_wrap is an enterprise only feature, you'll probably need to contruct kwargs and expand it here and only specify seal_wrap if true

Reply:
I think I see the issue now ... It isn't that it is an enterprise feature, it is that I was misusing the kvv2.configure method in hvac. That method is used for the kv specific configurations of the kv v2 secret engine. Adding [link to the hvac method](https://github.com/hvac/hvac/blob/9befaa4be80c88aa92c0ab69ba480d7c1d4009e5/hvac/api/secrets_engines/kv_v2.py#L18) for anyone who might run into this in the future. Thanks for your patience; still getting familiar with the code base :)